src/common/ipfixlolib/ipfixlolib.c

```diff
@@ -1316,9 +1316,6 @@ static int valid_transport_protocol(enum ipfix_transport_protocol p) {
 #ifdef IPFIXLOLIB_RAWDIR_SUPPORT 
```

Comment:
Should we move the `#ifdef` after the `case RAWDIR` and leave the `#else` part intact? Then it would be similar to the SUPPORT_SCTP ifdef?